src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/PersistorBase.java

```diff
@@ -97,6 +99,11 @@ public synchronized String read() {
     return jsonResponse;
   }
 
+  public synchronized String readRca(String rca) {
+    String jsonResponse = readRcaTable(rca);
```

Comment:
we can just have the return here without the variable name ?

Reply:
Done